src/test/resources/feature/BDDscenarioForWishList.feature

```diff
@@ -0,0 +1,36 @@
+@Artiom
+Feature: Wish List
+
+  Background:
+    Given user is on landing page
+
+  Scenario Outline: Create Wish List
+    Given user is logged in
```

Comment:
If you user is already on home page (as you write in Background) there is no need to logging in all over three scenarios.

Reply:
done